doc/api/fs.md

```diff
@@ -430,8 +430,11 @@ included in the iteration results.
 added: v10.10.0
 -->
 
-A representation of a directory entry, as returned by reading from an
-[`fs.Dir`][].
+A representation of a directory entry, which can be a file or a subdirectory
+within the directory, as returned by reading from an [`fs.Dir`][]. The
+directory entry is a structure of a mapping of a filename to an inode, which
+is unique for each file. The inode contains all information except the name
+and actual data of the file.
```

Comment:
While the information about `inode`s here is correct, I’m not sure whether it’s helpful to include it here:

- The name `inode` is OS-dependent, Windows doesn’t use it.
- While the inode does contain the information that is not the filename or the actual data, we don’t expose most of its data through `dirent`s and only provide it through `fs.stat` and similar calls. And conversely, a `dirent` does contain the filename, unlike the inode.

For our purposes, a directory entry is only a combination of the file name + file type pairs in a directory.

Reply:
Should we just keep the part about the mapping of the file to the inode then? Or just the part about the file name + file type?